js/src/components/shipping-rate-section/minimum-order-card/changeGroup.test.js

```diff
@@ -34,13 +34,88 @@ describe( 'getMinimumOrderChangeHandler( value, onChange, oldGroup )( newGroup )
 			},
 		},
 	] );
-	const mockOnChange = jest.fn();
+	// Pure add.
+	it( "returns new value with `newGroup.country`'s `shippingRate.options.free_shipping_threshold` updated to `newGroup.threshold`, when falsy `oldGroup` is given", () => {
```

Comment:
❓/ 💅   Doesn't matter if `oldGorup` is falsy because value contains the newGroup.